base/Dockerfile

```diff
@@ -12,10 +12,14 @@ RUN conda install --yes \
     nomkl \
     numpy==1.16.2 \
     pandas==0.24.2 \
-    && conda clean -tipsy
+    && conda update conda -y \
+    && conda clean -tipsy \
+    && find /opt/conda/ -type f,l -name '*.a' -delete \
+    && find /opt/conda/ -type f,l -name '*.pyc' -delete \
+    && find /opt/conda/ -type f,l -name '*.js.map' -delete \
```

Comment:
This seems like a good idea as I'm guessing people are not debugging JavaScript code in the container. Would encourage you to [raise an issue in conda-forge]( https://github.com/conda-forge/conda-forge.github.io/issues ) about doing this in all cases. There may be some people that want these for debugging, in which case we can split them out. However if no one wants these files, we could start just removing them.